src/Integration.test.js

```diff
@@ -226,4 +234,76 @@ describe('integration testing', () => {
       expect(healthScore > afterPlayHealth[index]).toBe(true)
     })
   })
+
+  it('presents a game message when the player runs out of food', () => {
+    toNaming()
+    toPacking()
+    toPlaying()
+    const mockPlayEvent = {target: {value: '1'}, keyCode: 13}
+    const playInput = app.find('#play')
+    playInput.simulate('keyDown', mockPlayEvent)
+    const gameMessage = app.find('#gameMessage').text()
+    expect(gameMessage).toBe('You have run out of food.')
+  })
+
+  it('changes the status of one person to "dead" when any number of health scores reaches 0 or less', () => {
+    toNaming()
+    toPacking()
+    toPlaying()
+    onPlay()
+    onPlay()
+    onPlay()
+    onPlay()
+    onPlay()
+    const people = app.state().people
+    const statusDead = people.filter((personObject) => personObject.status === 'dead')
+    expect(statusDead.length).toBe(1)
+  })
+
+  it('presents a game message when the status of a character changes to dead', () => {
+    toNaming()
+    toPacking()
+    toPlaying()
+    onPlay()
+    onPlay()
+    onPlay()
+    onPlay()
+    onPlay()
+    const gameMessage = app.find('#gameMessage').text()
+    expect(gameMessage.length > 0).toBe(true)
+  })
+
+  it('changes the state game gameState to "gameover" when the "You" character has a status of "dead"', () => {
+    toNaming()
+    toPacking()
+    toPlaying()
+    onPlay()
+    onPlay()
+    onPlay()
+    onPlay()
+    onPlay()
+    onPlay()
+    onPlay()
+    onPlay()
+    onPlay()
```

Comment:
Similarly, `onPlay()` 9 times results in `You` death? Seems a little brittle a test, doesn't it?

Reply:
Yes.  It's garbage.  I'll get some advice on how to test it tonight.